package.json

```diff
@@ -30,7 +30,9 @@
     "grunt-contrib-watch": "~0.4.0",
     "grunt-umd": "^2.3.3",
     "load-grunt-tasks": "^3.1.0",
+    "lodash": "~3.10.1",
     "qunitjs": "^1.17.1",
-    "video.js": "^4.12.4"
+    "sinon": "~1.16.1",
```

Comment:
I thought it was `~`.

I've run into a lot of issues in the past with `^` bringing in breaking changes (despite semver saying minor versions should be backward-compatible); sometimes multiple dependency layers deep.

That said, I'm fine with changing these. It ought to be a company/project-wide standard, though.